src/main/java/com/spotify/docker/BuildMojo.java

```diff
@@ -531,9 +548,9 @@ private void createDockerFile(String directory, List<String> filesToAdd) throws
     Files.write(Paths.get(directory, "Dockerfile"), commands, UTF_8);
```

Comment:
@dflemstr Does this line compile for you? Paths.get() expects a Paths object or Strings but directory is now a Path object.